README.md

```diff
@@ -213,6 +213,7 @@ conda install -c pytorch magma-cuda121  # or the magma-cuda* that matches your C
 
 # (optional) If using torch.compile with inductor/triton, install the matching version of triton
 # Run from the pytorch directory after cloning
+# If you need Intel GPU support, please make sure `export USE_XPU=1`
```

Comment:
Please refine the message a little bit.

Reply:
Thanks for the review! Updated in https://github.com/pytorch/pytorch/pull/126513/commits/f1728742ff2a364935362d90187c952ba53fa16b